sensor_msgs/Image face_image
---
string face_label
